lib/Doctrine/ruleset.xml

```diff
@@ -168,7 +168,7 @@
         <properties>
             <property name="allowFallbackGlobalConstants" type="boolean" value="false"/>
             <property name="allowFallbackGlobalFunctions" type="boolean" value="false"/>
-            <property name="allowFullyQualifiedGlobalClasses" type="boolean" value="false"/>
+            <property name="allowFullyQualifiedGlobalClasses" type="boolean" value="true"/>
```

Comment:
As an alternative I could live with treating global constants and classes the same (either way).

Reply:
Global functions/constants is not really viable as it creates an absolutely mind-boggling amount of inline diffs.